src/sparse_ci/ui64_determinant.h

```diff
@@ -152,6 +175,7 @@ uint64_t lowest_one_idx(uint64_t x);
 uint64_t clear_lowest_one(uint64_t x);
 
 double ui64_slater_sign(uint64_t x, int m, int n);
+double ui64_slater_sign(uint64_t x, int m);
```

Comment:
This is in part my fault. Can you please comment explaining the difference between these two functions?